FinniversKit/Sources/DNA/Color/ColorProvider.swift

```diff
@@ -174,4 +175,8 @@ public struct DefaultColorProvider: ColorProvider {
     public var borderDefault: UIColor {
         return .dynamicColor(defaultColor: .blueGray300, darkModeColor: .darkSardine)
     }
+
+    public var marketplaceNavigationBarIcon: UIColor {
+        .aqua400
+    }
```

Comment:
No dark mode?

Reply:
![image](https://github.com/finn-no/FinniversKit/assets/23578783/1403e700-e2ef-49f4-8d1c-c8c68bce9d51)
![image](https://github.com/finn-no/FinniversKit/assets/23578783/65375d1e-0ec8-4446-b3a7-0814eac83e15)
seems to be the same in both dark/light mode? (top icons, i.e "torget", "jobb" etc)